kubernetes/linux/main.sh

```diff
@@ -74,6 +121,15 @@ setCloudSpecificApplicationInsightsConfig() {
             echo "export APPLICATIONINSIGHTS_ENDPOINT=$APPLICATIONINSIGHTS_ENDPOINT" >>~/.bashrc
             source ~/.bashrc
             ;;
+         "bleu")
+            # Bleu cloud is a new cloud environment, we dont have AI in this cloud yet so using Public cloud for now.
+            # update this once we have AI in Bleu cloud.
```

Comment:
Since the "bleu" cloud currently uses public cloud Application Insights endpoints, are there any compliance or data residency concerns? If so, please document this in the code as a TODO or warning.

Reply:
Bleu is not ready yet and there are no customers. As I mentioned in comments, we would need to update to AI resource corresponding to Bleu cloud when its available.